workshops/find_bigfoot/README.md

```diff
@@ -229,6 +229,106 @@ Big Foot should now load after the background. We did it!
 
 Feel free to experiment with the `left` and `top` values to find a good place to hide Bigfoot.
 
+## Making The Game Harder
+
+Now we have another problem. This game is...too easy. Once somebody plays it once, they’ll be able to beat it over and over again with no challenge! We need to write a function that will move bigfoot randomly around the page after you beat it.
+
+Open your script.js file and let’s make a new function `moveBigFoot();`
+```js
```

Comment:
I think we might need an empty line between the text and the code block? Not sure actually.